front/src/applications/operationalStudies/views/Study.tsx

```diff
@@ -74,43 +71,69 @@ function displayScenariosList(
 export default function Study() {
   const { t } = useTranslation('operationalStudies/study');
   const { openModal } = useModal();
-  const [project, setProject] = useState<ProjectResult>();
-  const [study, setStudy] = useState<StudyWithFileType>();
   const [scenariosList, setScenariosList] = useState<ScenarioResult[]>([]);
   const [filter, setFilter] = useState('');
   const [filterChips, setFilterChips] = useState('');
   const [sortOption, setSortOption] = useState<SortOptions>('LastModifiedDesc');
-  const [studyStates, setStudyStates] = useState<StateType[]>([]);
   const dispatch = useDispatch();
-  const navigate = useNavigate();
   const projectId = useSelector(getProjectID);
   const studyId = useSelector(getStudyID);
-  const {
-    data: currentProject,
-    isLoading: isProjectLoading,
-    isError: isProjectError,
-    error: projectError,
-  } = osrdEditoastApi.useGetProjectsByProjectIdQuery({ projectId: projectId as number });
-  const [getCurrentStudy] = osrdEditoastApi.useLazyGetProjectsByProjectIdStudiesAndStudyIdQuery();
+
+  const { project, isError: isProjectError } = osrdEditoastApi.useGetProjectsByProjectIdQuery(
+    { projectId: projectId as number },
+    {
+      selectFromResult: (response) => ({
+        ...response,
+        project: response.data,
+      }),
+    }
```

Comment:
```suggestion
    {
      skip: !projectId,
      selectFromResult: (response) => ({
        ...response,
        project: response.data,
      }),
    }
```
We should also skip this call if there is no projectId